FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,56 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] in
+			switch $0 {
```

Comment:
Instead of using shorthand argument names, we can improve readability bu using named arguments:
```suggestion
		client.get(from: url) { [weak self] httpResult in
			switch httpResult {
```